htdocs/disableaccount.php

```diff
@@ -38,7 +44,7 @@
 }
 
 if ($audit_log_file) {
-    auditlog($audit_log_file, $dn, $audit_admin, "disableaccount", $result);
+    auditlog($audit_log_file, $dn, $audit_admin, "disableaccount", $result, $comment);
```

Comment:
Comment modal is only defined for now in lock/unlock panel in the HTML template.

We may add comment for all action, but this should maybe done in a separate PR.

Reply:
Maybe I'll do another PR for the comment feature...

Anyway, the last argument of the function auditlog is needed, else Service-Desk crashes.